composer.json

```diff
@@ -12,6 +12,10 @@
         "components/handlebars.js": "~3.0.0",
         "symfony/symfony": "~2.6"
     },
+    "require-dev": {
+        "ezsystems/ezpublish-kernel": "~6.0.0@beta | ~5.1",
```

Comment:
You are thinking about Symfony 2.6 here right? cause there is no hard dependency on kernel feature added in 5.4.1 and higher right?